client/src/legacy/LeftAndMain.EditForm.js

```diff
@@ -121,7 +121,7 @@ $.entwine('ss', function($){
           'There are validation errors on this page, please fix them before saving or publishing.'
         );
 
-        const $editFormErrorBanner = $("#Form_EditForm_error");
+        const $editFormErrorBanner = $("#Form_EditForm_error, #Form_ItemEditForm_error");
```

Comment:
Would be better to not rely on IDs but I'm going for the minimally invasive solution here, and `GridField`'s edit form always has the same ID.